### ISSUE TITLE
Issues with reproduction

### ISSUE THREAD

**Issue by Olamyy** [Niger-Volta-LTI/yoruba-adr#15]
I'm having some challenges reproducing the code on my local machine. The issue seems to be an error with torchtext but I honestly can't seem to figure out what exactly is causing it.

Here's a stacktrace.
<img width="1649" alt="Screen Shot 2019-05-14 at 4 23 39 PM" src="https://user-images.githubusercontent.com/14059714/57710410-d262ac80-7664-11e9-9bdb-0d3054a6bb7a.png">

To confirm it's not an environment issue, I tried running on google colab too but the same issue comes up, so it doesn't seem to be a versioning issue.

<img width="1672" alt="Screen Shot 2019-05-14 at 4 19 10 PM" src="https://user-images.githubusercontent.com/14059714/57710506-fcb46a00-7664-11e9-9ab5-7b2d1d61e022.png">

Do you have any idea what I might be missing?

**Comment by ruohoruotsi** [comment]
It appears that I did not execute the environment check (the first cell) is the [latest version of the Juptyer Notebook ](https://github.com/Niger-Volta-LTI/yoruba-adr/commit/b5d293c9d759458f7fdac69821ec09ac26209751#diff-03056fd8f5ffdcbf909ed0377e96967f).  The first cell typically prints out the environment so we can eliminate that from debugging. I typically use python 3.6 (see below) and have tested the code on python 3.5 (David uses 3.5). I have **NOT** tested the code with 3.7:
```
 "3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37) \n",
 "[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]\n"
```

I see from your first screencapture that you're using python3.7, but in the collab screencap is 3.6. So the problem is invariant to Python version 🤔  Okay, I suspect something is wrong with the model you have downloaded or your version of torch is incompatible/too-old since it's complaining it doesn't have an attribute that was added in October, 2018 (`_split_tokenizer`) 

* Can you please supply following output debug information:
```
$ python
Python 3.7.3 (default, Mar 27 2019, 16:54:48)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> print(torch.__version__)

* From the top level directory, please do
`$ls -alrt ./models` so I can see the size of the model itself.

**Comment by Olamyy** [comment]
Output in the image below

<img width="1680" alt="Screen Shot 2019-05-15 at 10 18 42 AM" src="https://user-images.githubusercontent.com/14059714/57764170-d93bfe80-76fa-11e9-8718-d3ae6e014434.png">

**Comment by ruohoruotsi** [comment]
okay, I'm back on my home machine now, I see that I'm using `1.0.0`
```
$ python
Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> print(torch.__version__)
1.0.0
>>>
```

[PyPI ](https://pypi.org/project/torch/#history/) states that `1.1` was just released [April 30th, ](https://github.com/pytorch/pytorch/releases)and this model was built and tested with `1.0`. A dot change usually doesn't introduce breaking changes, but who knows 🤷‍♂. 

Before I go debugging further, do you have a way to test it out with pytorch `1.0`?. This will help me elminate pytorch versions from the list of suspects.

On my side, when I reach the office this afternoon, I have a more current environment there, I'll see if I can reproduce this issue.

**Comment by ruohoruotsi** [comment]
Okay, I have eliminated `pytorch  1.1` as the culprit. Going deeper, as you stated originally, the problem is with `torchtext`. My working system shows I'm using version `0.4.0`
```
$ pip freeze | grep torchtext
torchtext==0.4.0
```

I will bet that your system says `torchtext==0.3.1`.  

If you look at the `requirements.txt` in the top level of this repo, you have to do:
 `pip install git+https://github.com/pytorch/text`. 

I typically just do `pip install -r requirements.txt` to just collect all the the correct versions of dependencies. Unfortunately, `pip install torchtext` will only get you `0.3.1`.   

This is the issue that you have been running into:  https://github.com/pytorch/text/issues/517

To summarize: 

-  keep your version `pytorch1.1`
- do `pip install git+https://github.com/pytorch/text`
- let me know the next errors 😬 😅

**Comment by ruohoruotsi** [comment]
Updated the README to clarify dependencies. `torchtext` is a bit tricky since it does NOT follow the usual `pip install torchtext` formula (for now)
